app/app_software.go

```diff
@@ -1,9 +1,12 @@
-//go:build ci
+//go:build ci || software
 
 package app
 
 import (
+	"image"
+
 	"fyne.io/fyne/v2"
+	software2 "fyne.io/fyne/v2/driver/software"
```

Comment:
I'd probably switch the naming here as the first of the two is the one called "software" in the public API